canonical/canonical.com
# Update navigation to use latest vanilla pattern

## Done

- Update navigation to use [latest vanilla pattern](https://vanillaframework.io/docs/patterns/navigation#dropdown)
- Removes some unused classes

## QA

- Check out this feature branch or https://canonical-com-836.demos.haus/
- Check that the code and visuals match that on vanilla (note: there are no sub-lists within vanilla so this couldn't be copied)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3145
